.changeset/neat-elephants-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-actions': patch
+---
+
+`yarn start` now starts a functional dev environment using a mock GitHub API to properly render the `<EntityGithubActionsContent />` component.
```

Comment:
Should this changeset be removed given that the PR does not actually alter any user-facing `github-actions` functionality?

Reply:
yep that's correct!

Reply:
Thanks @freben - I've removed the changeset!